plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/behavior/OpenShiftLaunchController.java

```diff
@@ -118,6 +116,7 @@ public void launch(ILaunchConfiguration configuration, String mode,
         launch.addProcess(new ServerProcess(launch, server, getLabel(launch.getLaunchMode())));
 		beh.setServerStarting();
 		waitForDeploymentConfigAndPods(server, monitor);
+		enableDevModeForNodeJsProject(OpenShiftServerUtils.getDeploymentConfig(server), server);
```

Comment:
this is the original code that was in place before it was completely removed. 
I believe that changing the dc is the right choice since every rc would be derived from it.
How likely is it that there's no dc but only a rc?